src/webserver/WsBase.h

```diff
@@ -320,9 +321,28 @@ class WsSession : public WsBase,
 
             return sendError(RPC::Error::rpcINTERNAL);
         }
-
+        auto result = response["result"].as_object();
+        result["warning"] = boost::json::array{};
         std::string responseStr = boost::json::serialize(response);
-        dosGuard_.add(*ip, responseStr.size());
+        auto warningFlag = false;
+        if (!dosGuard_.add(*ip, responseStr.size()))
+        {
+            result["warning"].as_array().emplace_back("Too many requests");
+            warningFlag = true;
+        }
+        auto lastPublishAgeMinutes =
```

Comment:
We may want to consider putting this as a member in `ReportingETL` so we don't have to do this `duration_cast` everywhere we want this info from the etl. 

```cpp
std::uint32_t 
ReportingETL::lastPublishAgeSeconds()
{
    return std::chrono::duration_cast<std::chrono::minutes>(
                 std::chrono::system_clock::now() - getLastPublish())
                 .count();
}
```

If you chose to go this route, also call this in `getInfo()`